src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
```

Comment:
Is this only needed in tests? If so, why not calling directly `enc.encode(input)`